### ISSUE TITLE
Add audible message when links are successfully inserted

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#2105]
Splitting this out from #2098

#2098 is going to make the Links suggestion accessible, replicating the current core implementtion: ARIA combobox + listbox.

<img width="505" alt="screen shot 2017-07-31 at 14 24 45" src="https://user-images.githubusercontent.com/1682452/28777584-16d76416-75fc-11e7-84e3-33c2438edd4f.png">

However, in the current core implementation (see `wp-includes/js/tinymce/plugins/wplink/plugin.js`) a message `Link inserted.` is used when the link is successfully inserted.

I guess this should be done in the other components every time a link is successfully inserted: Enter on the field, click on the button, click/Enter on the highlighted option.

Also, in the current core implementation the highlighted link gets "selected" when pressing the Tab key to move focus to the apply button:

<img width="453" alt="screen shot 2017-07-31 at 14 21 14" src="https://user-images.githubusercontent.com/1682452/28777492-a806c838-75fb-11e7-93a1-3cee54fbd417.png">

I'm not sure if Gutenberg should replicate this behavior, if so it should also use an audible message `Link selected.`
